### DISCUSSION TITLE
Running in VScode

### DISCUSSION THREAD

**Discussion by davidcollini** [snuffyDev/Beatbump#48] [Q&A]
So I downloaded VScode with the seventeenth extension as well as svelte intellisense, and I wanna make a few minor changes, but when I try to press F5 to run Beatbump, it doesn’t seem to work, I’m not sure what I’m doing wrong or is there a better way to run/bug test that I can do?

**Comment by snuffyDev** [comment]
Honestly that's something I need to improve upon, I'll see what I can do since I'm doing a full rewrite essentially. Should be able to integrate more with the debugger, especially since presently as you mentioned there's not much in the way of it.

Apart from that though, the way I normally run it is just using `npm run dev` -- but I remember you mentioning you have an Apple computer of some sorts, which could potentially be a port issue.

Try changing the port in `package.json` under the "dev" script to something else, or just change the command to `svelte-kit dev --host`. Let me know if you have any other issues with it
